src/libs/ReportUtils.ts

```diff
@@ -503,12 +503,6 @@ Onyx.connect({
     callback: (value) => (allReports = value),
 });
 
-let doesDomainHaveApprovedAccountant = false;
```

Comment:
Do you think we should remove `doesDomainHaveApprovedAccountant` from the BE now? 🤔 cc @yuwenmemon

Reply:
Yeah if we're not using it in the front-end anymore I can certainly rip it out of the back end!

Reply:
It originally came from: https://github.com/Expensify/Expensify/issues/257570

I presume we're just showing domain rooms universally now?

Reply:
No, we're hiding them from approved accountants: https://expensify.slack.com/archives/C03SSAQ7P/p1713199776085039

Reply:
Ah, same same for this though! I created an issue here so I don't forget: https://github.com/Expensify/Expensify/issues/389013

Reply:
ice, thanks!